### PR TITLE
Set up code owners

### PR DESCRIPTION
**Pull request by swissspidy**
Adds certain people as code owners for certain areas.

See https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners for documentation.

See #319.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (spacedmonkey) [inline .github/CODEOWNERS]: We could also add 
`assets/src/edit-story/app/api/                @spacedmonkey`

File: .github/CODEOWNERS
Comment (spacedmonkey) [inline .github/CODEOWNERS]: `docs/                                   @swissspidy @pbakaus`

File: .github/CODEOWNERS
Comment (spacedmonkey) [inline .github/CODEOWNERS:18]: `*.js @swissspidy @dvoytenko barklund`

File: .github/CODEOWNERS
Comment (swissspidy) [reply .github/CODEOWNERS]: Hesitating with that one as it will contain mostly developer documentation.

File: .github/CODEOWNERS
Comment (spacedmonkey) [inline .github/CODEOWNERS]: `.travis.yml @swissspidy @spacedmonkey`

File: .github/CODEOWNERS
Comment (swissspidy) [reply .github/CODEOWNERS]: Skipping for now.

File: .github/CODEOWNERS
Comment (barklund) [inline .github/CODEOWNERS]:
```suggestion
assets/src/edit-story/                          @swissspidy @dvoytenko @barklund
```

File: .github/CODEOWNERS
Comment (barklund) [inline .github/CODEOWNERS]: Perhaps test infrastructure too, as they don't contain the actual tests, just the setup?

```
tests/                                            @swissspidy
```

File: .github/CODEOWNERS
Comment (spacedmonkey) [inline .github/CODEOWNERS:12]: Should we set @miina as owner of movable?

File: .github/CODEOWNERS
Comment (swissspidy) [reply .github/CODEOWNERS]: Not necessarily true for e2e tests and PHP unit tests

**Comment by swissspidy** [comment]
^ @barklund

**Comment by barklund** [comment]
@googlebot I fixed it.

**Comment by swissspidy** [comment]
OK well apparently it didn't get it 🙄 Will add it manually at the end.

File: .github/CODEOWNERS
Comment (swissspidy) [reply .github/CODEOWNERS:12]: Thoughts, @miina?

Since Movable usage is split across a few individual files and folders, we'd just have to make sure the list here in this file is updated accordingly when refactoring etc.

Comment (miina) [reply .github/CODEOWNERS:12]: Thanks for tagging this, @spacedmonkey 

Ideally yes, would be great.
Q: We still follow the idea of requesting a review from everyone in case of JS-related PR-s, correct?
If yes, then it's fine if some file is missed for the automatic review request with refactoring later, it's OK to add just the main movable files for now (in the `movable` folder + the 2 files in the `canvas` folder: `multiSelection-` and `singleSelectionMovable`) .

Comment (swissspidy) [reply .github/CODEOWNERS:12]: This change would just automatically add code owners as reviewers, but of course one can & should still add more reviewers.

In the future, we could think about requiring reviews from code owners.

**Comment by swissspidy** [comment]
@dvoytenko @barklund @spacedmonkey @miina WDYT about the proposed code owners config?  Potentially noisy or OK?

**Review by miina — APPROVED** [review]
Looks OK to me.

**Review by barklund — APPROVED** [review]
I'm fine with this. Let's try it out, and if it becomes too noisy, we can always change it.

I really like the "default reviewer" thing - especially for a growing team.

**Review by dvoytenko — APPROVED** [review]

**Comment by dvoytenko** [comment]
LGTM from me, but I believe you should make @pbakaus a global owner as well given the time difference. I'm sure he won't abuse it. Plus, two global owners is definitely better than one.

**Pull request merged**
